mysql/const.go

```diff
@@ -44,6 +44,13 @@ const (
 	ServerPSOutParams              uint16 = 0x1000
 )
 
+// Server limitations
+const (
+	ServerTableNameLength    int = 64
```

Comment:
The limitation has nothing to do with server, name it 'MaxTableNameLength` is better.